src/core/db/manager.py

```diff
@@ -237,8 +279,8 @@ def list_repo_files(self, repo):
 
     def list_repo_cards(self, repo):
         # check for permission
-        res = DataHubManager.has_repo_privilege(
-            self.username, self.repo_base, repo, 'USAGE')
+        res = DataHubManager.has_repo_file_privilege(
```

Comment:
Cards are a datahub-ism, rather than a database-ism. Checking for 'Usage' didn't actually let us know whether a user should be able to execute the card's query or not... and trying to execute the card's query on behalf of the user could have unintended consequences. (For example, if the card contained an insert statement). Given that cards are closer to a file than a database construct, it seemed to make more sense to tie cards to file permissions.